assets/src/scss/base/_mixins.scss

```diff
@@ -317,3 +317,88 @@ $nested-comment-left-margin: 50px;
   -webkit-line-clamp: $lines;
   overflow: hidden;
 }
+
```

Comment:
I move all the related code into a `mixin`. Why? To have a consistent layout between forms (P4 default and GF).